nginx.conf

```diff
@@ -0,0 +1,35 @@
+worker_processes auto;
+pid /run/nginx.pid;
+error_log /var/log/nginx/error.log;
+include /etc/nginx/modules-enabled/*.conf;
+
+events {
+  worker_connections 768;
+}
+
+http {
+  sendfile on;
+  tcp_nopush on;
+  types_hash_max_size 2048;
+
+  include /etc/nginx/mime.types;
+  default_type application/octet-stream;
+
+  ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3; # Dropping SSLv3, ref: POODLE
+  ssl_prefer_server_ciphers on;
+
+  access_log /var/log/nginx/access.log;
+  gzip on;
+  include /etc/nginx/conf.d/*.conf;
+  server {
+    listen 8080;
+
+    index index.html index.htm index.nginx-debian.html;
```

Comment:
stick with just one, that your build produces. No need to solve million of use cases.

Reply:
Left only index.html